.github/workflows/inactive-issues.yaml

```diff
@@ -0,0 +1,22 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: -1
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 30 days with no activity."
```

Comment:
Will this appear when I add the "stale" label manually?

Reply:
Unsure 😕

Reply:
unsure..  leaning to the answer no

Reply:
I can see it being slightly annoying if we had to manually write this every time (it would be good to let the user know their issue has become stale / how much time they have to respond)